src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -434,6 +434,15 @@ Public Class BuildDevDivInsertionFiles
         GenerateTestFileDependencyList(NameOf(IntegrationTestFilesExtra), IntegrationTestFilesExtra, insertedFiles)
     End Sub
 
+    Private Function GetPathRelativeToBinaries(p As String) As String
+        Debug.Assert(p.StartsWith(_binDirectory, StringComparison.OrdinalIgnoreCase))
```

Comment:
I'd say no debug assert here. The generate solution code had this same pattern and it was terrifying (partly because the function was actually broken.)